adobe_vipm/flows/sync/agreement.py

```diff
@@ -1077,10 +1084,19 @@ def sync_agreement(
         return
 
     adobe_customer_id = flows_utils.get_adobe_customer_id(agreement)
-    customer = get_customer_or_process_lost_customer(
+    if not adobe_customer_id:
+        message = (
+            f"CustomerId not found in Agreement {agreement['id']} with params "
+            f"{agreement['parameters']}. Skipping."
+        )
+        logger.warning(message)
+        notify_agreement_unhandled_exception_in_teams(agreement["id"], message)
+        return
+
+    adobe_customer = get_customer_or_process_lost_customer(
         mpt_client, adobe_client, agreement, adobe_customer_id, dry_run=dry_run
     )
-    if not customer:
+    if not adobe_customer:
         return
```

Comment:
Could you pls, make a comment here, that agreement was processed correctly here, so we don't need to do it